prime-router/src/main/kotlin/Sender.kt

```diff
@@ -36,10 +36,12 @@ import java.time.OffsetDateTime
 @JsonTypeInfo(
     use = JsonTypeInfo.Id.NAME,
     include = JsonTypeInfo.As.EXISTING_PROPERTY,
-    property = "topic"
+    property = "topic",
+    visible = true
```

Comment:
`visible = true` is necessary to allow us to deserialize the `topic` value and not _just_ use it to determine the sender type.